docs/DeveloperGuide.md

```diff
@@ -677,29 +654,50 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
       Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
-### Deleting a student
-
-1. Deleting a student while all students are being shown
-
-   1. Prerequisites: List all students using the `list` command. Multiple students in the list.
-
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-   1. Test case: `delete 0`<br>
-      Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
-
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
-
-1. _{ more test cases …​ }_
-
-### Saving data
-
-1. Dealing with missing/corrupted data files
+### Using the application
+
+#### Student management
+1. Creating, removing, and viewing students
+   
+   1. Create a student with the name "John Doe" using `student add n/John Doe e/abc@example.com p/91234567`. <br>
+      Expected: The UI is updated with the new student list.
+   2. Edit the student with the command `student edit 4 p/97654321`
+   3. Delete the student with the command `student delete 4`
+
+#### Task management
+1. Creating, removing, grading and viewing tasks
+
+    1. Create a student with the name "Assignment 1" using `task add tn/Assignement 1 i/Complete Assignment 1 d/10/10/2022`. <br>
+    2. Click on the `Tasks` Button on the left side of the screen. <br>
+       Expected: The UI is updated with the new task list.
+    3. Edit the student with the command `task edit 1 s/David Li`
+    4. Double click on the task to view the task details.
+       Expected: The details shows the list of students assigned to the task.
+    5. Grade the student with the command `grade edit 1 1 gr/T`
+    6. Change the view back the students, then switch back to Tasks view, the task should be updated with the new grade. <br>
+**:information_source: This is a know bug with JavaFX**<br>
+    7. Delete the student with the command `task delete 1`
+
+#### Tutorial group management
+1. Creating and managing tutorial groups
+   1. Create a new tutorial group with the format TXX. `tutorial add g/T01` 
+   2. See a list of tutorial groups using `tutorial list` <br>
+      Expected: A list of tutorial groups is shown.
+   3. Add a student to the tutorial group with the format `student enroll 1 g/T01`
+   4. The UI should be updated with the new student list with update tutorial group info.
+   5. Filter the student list by tutorial group with the format `tutorial filter g/T01`
+   6. Clear the filter with the format `student unfilter`
```

Comment:
Need to add grades

Reply:
Added grade in the earlier manual testing case